src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Methodologies/MethodologyController.cs

```diff
@@ -21,53 +20,54 @@ public MethodologyController(IMethodologyService methodologyService)
             _methodologyService = methodologyService;
         }
 
-        [Produces("application/json")]
-        [HttpGet("api/topic/{topicId}/methodologies")]
-        public Task<ActionResult<List<MethodologyViewModel>>> GetTopicMethodologiesAsync([Required]Guid topicId)
-        {
-            return _methodologyService
-                .GetTopicMethodologiesAsync(topicId)
-                .HandleFailuresOrOk();
-        }
-        
         [Produces("application/json")]
         [HttpGet("api/methodologies")]
-        public Task<ActionResult<List<MethodologyViewModel>>> GetMethodologiesAsync()
+        public Task<ActionResult<List<MethodologyStatusViewModel>>> GetMethodologiesAsync()
```

Comment:
Shouldn't this be `MethodologySummaryViewModel`?

Reply:
It can easily be that but it's not at the moment because it's only used by views which are tables consisting of title, and status which is `MethodologyStatusViewModel`

Reply:
Changed to return `MethodologySummaryViewModel` to be consistent.